File: tokit/tokit/generator.cpp
///<------------------------------------------------------------------------------
//< @file:   tokit\tokit\generator.cpp
//< @author: 洪坤安
//< @date:   2014年9月25日 14:46:17
//< @brief:  
//< Copyright (c) 2014 Tokit. All rights reserved.
///<------------------------------------------------------------------------------

#include "generator.h"
#include "parser.h"
#include "file_tool.h"

void generator::echo_errs()
{
    echotool::echo_errvec(m_errvec);
    m_errvec.clear();
}

File: tokit_go_src/tokit/generator/generator.go
///<------------------------------------------------------------------------------
//< @文件名：generator.cpp
//< @作　者：洪坤安
//< @日　期: 2014年11月12日 17:31:42
//< @摘　要:
//< Copyright (c) 2014 Tokit. All rights reserved.
///<------------------------------------------------------------------------------

// generator
package generator

import (
	"tokit/tool"
)

type generator interface {
	Generate()
}

type generate struct {
	excel   string
	cfgbase Cfgbase_t
	errvec  tool.Errvec_t
}

func echo_errs() {

}
